code/datums/uplink_items/uplink_traitor.dm

```diff
@@ -557,9 +557,9 @@
 
 /datum/uplink_item/bundles_TC/badass
 	name = "Syndicate Bundle"
-	desc = "Syndicate Bundles are specialised groups of items that arrive in a plain box. These items are collectively worth more than 100 telecrystals, but you do not know which specialisation you will receive."
+	desc = "Syndicate Bundles are specialised groups of items that arrive in a plain box. These items are collectively worth more than 100 telecrystals, but you do not know which specialisations you will receive. You can select one our of three specialisations after purchase."
```

Comment:
"our" typo and i would edit this description even further since you no longer dont 100% know what specialisation you will receive.